### ISSUE TITLE
Some questions about the performance of AdvRush in your table.

### ISSUE THREAD

**Issue by MooMoo-Yang** [zhicheng2T0/Wsr-NAS#2]
The accuracy of AdvRush in your table is significantly lower than that in the original text **(53.03%)** (when the \epsilon is set to 0.03), and you did not cite its accuracy under FGSM attacks. Can you explain the reason for this? Additionally, could you provide detailed training parameters, including learning rate and training scripts? Thanks.
